client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -26,15 +35,67 @@ const MailboxListHeader = ( { children } ) => {
 	);
 };
 
-const MailboxListItem = ( { children, isPlaceholder, hasNoEmails } ) => {
+const MailboxListItem = ( { children, isError = false, isPlaceholder, hasNoEmails } ) => {
 	const className = classNames( 'email-plan-mailboxes-list__mailbox-list-item', {
 		'is-placeholder': isPlaceholder,
 		'no-emails': hasNoEmails,
 	} );
-	return <CompactCard className={ className }>{ children }</CompactCard>;
+	return (
+		<CompactCard className={ className } highlight={ isError ? 'error' : null }>
+			{ children }
+		</CompactCard>
+	);
+};
+
+const MailboxListItemSecondaryDetails = ( { children, className } ) => {
+	const fullClassName = classNames( 'email-plan-mailboxes-list__mailbox-secondary', className );
```

Comment:
For consistency, let's use a more meaningful class name e.g. `email-plan-mailboxes-list__mailbox-secondary-details`

Reply:
Done!